tpch/pandas_queries/queries.py

```diff
@@ -513,6 +513,7 @@ def q08(root: str, storage_options: Dict):
         (orders["O_ORDERDATE"] >= pd.Timestamp("1995-01-01"))
         & (orders["O_ORDERDATE"] < pd.Timestamp("1997-01-01"))
     ]
+    orders_filtered = orders_filtered.copy()
```

Comment:
Do we really need a copy()?

Reply:
Yes, it is indeed possible not to add it, but the compiler recommends it. For the sake of time testing, I will delete it.